static/ng/resources/js/components/summary/summary.directive.js

```diff
@@ -0,0 +1,45 @@
+(function() {
+    
+    'use strict';
+    
+    angular
+      .module('harbor.summary')
+      .directive('projectSummary', projectSummary);
+      
+    ProjectSummaryController.$inject = ['StatProjectService'];
+    
+    function ProjectSummaryController(StatProjectService) {
+        var vm = this;
+        
+        StatProjectService()
+          .success(statProjectSuccess)
+          .error(statProjectFailed);
+          
+        function statProjectSuccess(data, status) {
+            vm.statProjects = data;
+            if(vm.statProjects.hasOwnProperty("total_project_count")) {
```

Comment:
It's not a good idea to check a user whether he is a admin by checking a property.